packages/eas-cli/src/worker/assets.ts

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable async-protect/async-suffix */
```

Comment:
What is the reason for this? Do you find `Async` suffixes annoying?

Reply:
Oh sorry, I'll get that cleaned up.

On a side-note, I do actually find it redundant and annoying 😄 It was especially annoying in this case because of `packFilesIterable` which isn't even returning a promise, but an async generator. Anyway, I'll get this cleaned up 👀 Just forgot to do so. nice catch!